src/shared_modules/indexer_connector/src/indexerConnectorSyncImpl.hpp

```diff
@@ -473,10 +472,8 @@ class IndexerConnectorSyncImpl final
     }
     void bulkIndex(std::string_view id, std::string_view index, std::string_view data)
     {
-        constexpr auto FORMATTED_SIZE {20 + 8 + 2 + 2 + 1};
-        std::lock_guard lock(m_mutex);
-
-        if (m_bulkData.length() + FORMATTED_SIZE + index.size() + id.size() + data.size() > MaxBulkSize)
+        if (constexpr auto FORMATTED_SIZE {20 + 8 + 2 + 2 + 1};
```

Comment:
Couldn't this be more clear? what do represent those numbers?

Reply:
Don`t use magic number, define with `constexpr`